src/util/measure_detector_to_reference.py

```diff
@@ -0,0 +1,103 @@
+import asyncio
+import hjson
+from ipaddress import IPv4Address
+import json
+import logging
+import numpy as np
+import os
+from time import sleep
+from timeit import main
+
+
+from src.board_builder.board_builder import BoardBuilder
+from src.common.structures.component_role_label import COMPONENT_ROLE_LABEL_DETECTOR, COMPONENT_ROLE_LABEL_POSE_SOLVER
+from src.controller.mct_controller import MCTController
+from src.controller.structures.mct_component_address import MCTComponentAddress
+
+input_filepath = "/home/adminpi5/Documents/MCSTrack/data/measure_detector_to_reference_config.json"
```

Comment:
With a little bit more work, this could be turned into a real utility that can be run from the command line with an input. Have you ever used `argparse`, or even just `sys.argv`?